cli/commit_message.go

```diff
@@ -0,0 +1,58 @@
+package cli
```

Comment:
Let's create a /interface package and add this file to that so that it's not mixed in with the other command files.

Reply:
makes sense, will do